channels/OFF-CHAIN.md

```diff
@@ -1009,3 +1010,61 @@ contract update](../serializations.md#channel-off-chain-update-call-contract). I
 information needed for a contract call, including the contract address. The other participant co-signs
 the changes and the contract call is considered to be executed. Its results can be extracted from the calls tree in the state tree.
 
+### Contracts referring on-chain data
+
+Contracts being used in channels off-chain calls have the exact same semantics
+as one being used on-chain. Because of the different environment though, those
+might have different results as the on-chain ones.
+
+Since there is no signle source of truth, each participant considers one's chain
+to be the correct one. This is essential for the trustless environment. Every
+contract call is pinned to the top of the chain, as it is seen by each
```

Comment:
@knarz currently each participants treats its own chain as the only source of truth. No block hash is propageted between parties. This is to change as described by PT [SC calls: pin enviroment to a block hash](https://www.pivotaltracker.com/story/show/160843011) - each off-chain call will include the on-chain block hash it was based upon.
This will impact only off-chain FSM communication between parties and will have no impact on on-chain forcing of progress. In that case the miner's top block is considerered the pinned one.

Since this will not impact consus, it is marked as post main net.